api/main.go

```diff
@@ -41,60 +43,97 @@ func authGuard(core *Core) fiber.Handler {
 	return func(ctx *fiber.Ctx) error {
 		authHeader := ctx.Get("Authorization")
 		if authHeader == "" {
-			return ctx.Status(fiber.StatusUnauthorized).JSON(fiber.Map{
-				"error": "Missing Authorization header",
+			return ctx.Status(fiber.StatusUnauthorized).JSON(ErrorResponse{
+				http.StatusUnauthorized,
+				"Authorization header is required",
+				nil,
```

Comment:
I find it weird that you have to provied StatusUnauthorized twice (once in the `Status()` and once in the `JSON()` response.
I wonder if this can be replaced with only the second.

Reply:
Applies to the others as well.